pkgs/applications/audio/magnetophonDSP/VoiceOfFaust/default.nix

```
@@ -11,13 +11,13 @@
 }:
 stdenv.mkDerivation rec {
   pname = "VoiceOfFaust";
-  version = "1.1.5";
+  version = "V1.1.7";
 
   src = fetchFromGitHub {
     owner = "magnetophon";
     repo = "VoiceOfFaust";
     rev = version;
```

```suggestion
  version = "1.1.7";

  src = fetchFromGitHub {
    owner = "magnetophon";
    repo = "VoiceOfFaust";
    tag = "V${version}";
```
As far as I know the version must start with a digit

---

Thanks for the review, done!